dev-requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.11
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
In the `Dockerfile` we use 3.11, should this also be used to compile the requirements?